### DISCUSSION TITLE
[TEMPLATE] Minitel

### DISCUSSION THREAD

**Discussion by WITNicolas** [witsa/synapps#656] [Show and tell]
Pour fêter dignement la sortie officielle de STUDIO 1.2.0, voici un template IHM Minitel ... fonctionnel évidement ....
Il y a les effets de scanline qui clignotent bien comme il faut et qui fait mal aux yeux .... évidement
Et c'est en noir et blanc .... évidement....

edit : 
J'avais oublié de mettre le son ... c'est fait....

La fin d'une époque .... et bien non le retro a toujours le vent en poupe ! 🥇 

![image](https://github.com/witsa/synapps/assets/33632664/beeac2cb-6c1e-4b64-9eb3-16b10d7e7364)

![image](https://github.com/witsa/synapps/assets/33632664/3990d38a-ca2f-4f5a-a154-ddc58eac0073)

![image](https://github.com/witsa/synapps/assets/33632664/11be62b1-667b-4042-a306-38518dd8b57c)

[templateMinitel.zip](https://github.com/witsa/synapps/files/13356785/templateMinitel.zip)

**Comment by manufitoussiwit** [comment]
C'est tellement bien fait que j'entends le bruit de connexion 😂

**Reply by WITNicolas** [reply]
ha ben je viens de le rajouter, je savais que j'avais oublié quelquechose ..... 🤣

**Reply by FlorentAdell** [reply]
C'est trop bien fait !! MDR !!
